File: Python/Parqueadero.py
class Parqueadero:

    _id_contador = 0

    def __init__(self, nombre, area, cubiculo, direccion, estado):
        Parqueadero._id_contador+=1
        self.id = Parqueadero._id_contador
        self.nombre = nombre
        self.area = area
        self.cubiculo = cubiculo
        self.direccion = direccion
        self.estado = estado


    #Getters
    def get_id(self):
        return self.id
    
    def get_nombre(self):
        return self.nombre

    def  get_area(self):
        return self.area

    def get_cubiculo(self):
        return self.cubiculo
 
    def get_direccion(self):
        return self.direccion

    def get_estado(self):
        return self.estado

    # Setters

    def set_nombre(self, nombre):
        self.nombre = nombre
    
    def set_area(self, area):
        self.area = area

    def set_cubiculo(self, cubiculo):
        self.cubiculo = cubiculo
    
    def set_direccion(self, direccion):
        self.direccion = direccion

    def set_estado(self, estado):
        self.estado = estado

       

File: Java/Parqueadero.java


public class Parqueadero {


    private int id;
    private static int contadorId;
    private String nombre;
    private String area;
    private Cubiculo cubiculo;
    private String direccion;
    private String estado;

    public Parqueadero(String nombre, String area, Cubiculo cubiculo, String direccion, String estado){
        this.id = ++Parqueadero.contadorId;
        this.nombre = nombre;
        this.area = area;
        this.cubiculo = cubiculo;
        this.direccion = direccion;
        this.estado = estado;
    }

    public int getId() {
        return id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getArea() {
        return area;
    }

    public void setArea(String area) {
        this.area = area;
    }

    public Cubiculo getCubiculo() {
        return cubiculo;
    }

    public void setCubiculo(Cubiculo cubiculo) {
        this.cubiculo = cubiculo;
    }

    public String getDireccion() {
        return direccion;
    }

    public void setDireccion(String direccion) {
        this.direccion = direccion;
    }

    public String getEstado(){
        return this.estado;
    }

    public void setEstado(String estado){
        this.estado = estado;
    }

    


}